content/en/tracing/setup_overview/setup/dotnet-core.md

```diff
@@ -411,16 +411,18 @@ The following table below lists configuration variables that are available for b
-| `DD_TRACE_LOG_PATH`                                            | Sets the path for the automatic instrumentation log file and determines the directory of all other .NET Tracer log files. Ignored if `DD_TRACE_LOG_DIRECTORY` is set.                                                                              |
-| `DD_DISABLED_INTEGRATIONS`<br/><br/>`DisabledIntegrationNames` | Sets a list of integrations to disable. All other integrations remain enabled. If not set, all integrations are enabled. Supports multiple values separated with semicolons. Valid values are the integration names listed in the [Integrations][2] section.           |
+| `DD_TRACE_ENABLED`<br/><br/>`TraceEnabled`                      | Enables or disables all automatic instrumentation. Setting the environment variable to `false` completely disables the CLR profiler. For other configuration methods, the CLR profiler is still loaded, but traces will not be generated. Valid values are: `true` (default) or `false`. |
+| `DD_TRACE_DEBUG`                                                | Enables or disables debug logs in the Tracer. Valid values are: `true` or `false` (default). Setting this as an environment variable also enabled debug logs in the CLR Profiler.                                                                                                        |
+| `DD_TRACE_LOG_DIRECTORY`                                        | Sets the directory for .NET Tracer logs.<br/><br/>Windows default: `%ProgramData%\Datadog .NET Tracer\logs\`<br/><br/>Linux default: `/var/log/datadog/dotnet/`                                                                                                                                                                                     |
+| `DD_TRACE_LOG_PATH`                                             | Sets the path for the automatic instrumentation log file and determines the directory of all other .NET Tracer log files. Ignored if `DD_TRACE_LOG_DIRECTORY` is set.                                                                              |
+| `DD_DISABLED_INTEGRATIONS`<br/><br/>`DisabledIntegrationNames`  | Sets a list of integrations to disable. All other integrations remain enabled. If not set, all integrations are enabled. Supports multiple values separated with semicolons. Valid values are the integration names listed in the [Integrations][2] section.           |
+| `DD_TRACE_ADONET_EXCLUDED_TYPES`<br/><br/>`AdoNetExcludedTypes` | Sets a list of `AdoNet` types that will be excluded from automatic instrumentation. (e.g. : `DD_TRACE_ADONET_EXCLUDED_TYPES=System.Data.SqlClient.SqlCommand`) |
```

Comment:
Is there anything we can do to help with the UI table now being larger than all other tables? Looks like we might be able to add a \n to the (e.g and drop it? Or change the font size? 

Looks like this is on both .NET Framework and .NET Core.

Reply:
If we can't change the font size, we'd have to remove the example usage because the configuration name is so long. And whenever we use the `code element`, our rendering will not break it up

Reply:
Hi there! We're currently in the process of looking into how to resolve these issues in the near future. For the interim, you can use `{{< table table-type="break-word" >}}` or try to shorten type name as mentioned above. Example available here: https://github.com/DataDog/documentation/blob/master/content/en/agent/kubernetes/operator_configuration.md

Reply:
@jaycdave88 @sarina-dd updated above. I decided to just use the code element on the type name so we avoid the overflow. Let me know if it reads okay 😄

Reply:
LGTM